job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -61,6 +64,14 @@ object WebApi {
     Map(ResultKey -> result)
   }
 
+  def resultToByteIterator(jobReport: Map[String, Any], result: Iterator[_]): Iterator[_] = {
+    ResultKeyStartBytes.toIterator ++
+      (jobReport.map(t => Seq(AnyJsonFormat.write(t._1).toString(),
```

Comment:
Hmmm... why not just format the jobReport directly into a string, instead of doing it by hand?  WebApi elsewhere translates maps to strings. `AnyJsonFormat.write(jobReport)` should do it I think.